embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/config/behavior/SensitiveKeysBehaviorImpl.kt

```diff
@@ -0,0 +1,16 @@
+package io.embrace.android.embracesdk.internal.config.behavior
+
+import io.embrace.android.embracesdk.internal.config.local.SdkLocalConfig
+
+private const val SENSITIVE_KEY_MAX_SIZE = 128
+
+internal class SensitiveKeysBehaviorImpl(
+    private val localConfig: SdkLocalConfig
+) : SensitiveKeysBehavior {
+
+    // TODO: max list length
```

Comment:
`private val list = localConfig.sensitiveKeysDenylist.take(n)` would be a good option for this